proposals/stage-0/regular-town-halls/README.md

```diff
@@ -0,0 +1,55 @@
+# Regular Town Halls
+>  Stage 0
+
+## Champion
+
+[Myles Borins](https://github.com/MylesBorins)
+
+## Description
+
+A monthly / quarterly open Q/A between the Board of Directors of $FOUNDATION
+and the projects / community.
```

Comment:
I think quarterly would be best.

Reply:
I am not 100% on that timing tbqh. Would very much like to iron that out between stage 1 and 2